drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -871,6 +871,7 @@ function DragAndDropBlock(runtime, element, configuration) {
                 input: input,
                 displayName: item.displayName,
                 imageURL: imageURL,
```

Comment:
I found it confusing to use the property name `imageURL` for different things in different places, but I'm happy to change it.  The actual image URL is indeed not needed for the view; I just left it there for completeness.  I don't think this would cause any further bugs, since neither of the URLs are ever changed on the view, and the image URL is only used in a single place.

This comment made me realise that I need to move the fallback to the old attribute name to the Python code (since the URL gets expanded there).